### DISCUSSION TITLE
Why Refind?

### DISCUSSION THREAD

**Discussion by suvil** [Sithuk/ubuntu-server-zfsbootmenu#41] [General]
Naive question but why refind? ZBM docs state that UEFI can directly boot ZBM. So refind is there only to increase compatibility with non-UEFI or half-baked UEFI systems?

**Comment by Sithuk** [comment]
See the zfsbootmenu team's response to that query at the link below.
https://docs.zfsbootmenu.org/en/v2.3.x/general/uefi-booting.html#avoiding-an-intermediate-boot-manager
